pages/queries.rst

```diff
@@ -45,6 +45,10 @@ Messages that do not have the field *type*::
 
   NOT _exists_:type
 
+Messages that match RegExp *FastEthernet[0-9]+* (ElasticSearch will convert messages to lowercase)::
```

Comment:
In fact, Elasticsearch will only convert the query to lower case, if the configured tokenizer does that:
https://www.elastic.co/guide/en/elasticsearch/reference/5.6/analysis-lowercase-tokenizer.html

For example, the standard analyzer of Elasticsearch is using the lowercase tokenizer:
https://www.elastic.co/guide/en/elasticsearch/reference/5.6/analysis-standard-analyzer.html

Reply:
But if I search `FastEthernet[0-9]+`, the result is empty, if I search `fastethernet[0-9]+`, the result is normal, may be I just use all lowercase in this example.

Reply:
As explained, it depends on the tokenizer which is being used for the respective field(s).

Maybe add examples and an explanation for when lowercase searches work and when not.

Reply:
Actually I don't know what is tokenizer? 😄 I just know the lowercase works...

Reply:
Can you help to add the explaination? 😃